db/collection.go

```diff
@@ -236,6 +236,7 @@ func (db *db) updateCollection(
 	if err != nil {
 		return nil, err
 	}
+	priorSchemaVersionID := desc.Schema.VersionID
```

Comment:
suggestion: This could probably be defined closer to where it is used bellow.

Reply:
`desc.Schema.VersionID` gets overwritten two lines down, it needs to be here (or one line down)

Reply:
Oh yes I missed that sorry.